config.json

```diff
@@ -3,7 +3,7 @@
   "slug": "haskell",
   "active": true,
   "status": {
-    "concept_exercises": false,
+    "concept_exercises": true,
```

Comment:
Please don't edit this value. Changing this to `true` activates "Learning Mode" on the website, and should only be done by track maintainers or Exercism staff members when we're ready to launch this feature for the track.

Reply:
Reverted.